akka-actor-tests/src/test/scala/akka/actor/CoordinatedShutdownSpec.scala

```diff
@@ -366,13 +382,7 @@ class CoordinatedShutdownSpec extends AkkaSpec {
     def systemConfig: Config
     def withSystemRunning(system: ActorSystem): Unit
 
-    val newSystem = ActorSystem(
-      systemName,
-      ConfigFactory.parseString(
-        """
-          akka.coordinated-shutdown.run-by-jvm-shutdown-hook = on
-          akka.coordinated-shutdown.terminate-actor-system = on
-        """))
+    val newSystem = ActorSystem(systemName, systemConfig)
```

Comment:
Oh wait, so the config was actually not used in the tests!?!

Reply:
yup